tests/conftest.py

```diff
@@ -68,6 +68,7 @@ def app(request):
     Babel(app)
     Mail(app)
     InvenioDB(app)
+    app.debug = True
```

Comment:
Aaand nevermind what I wrote. Turns out `debug` wasn't needed at all.
Disabling `LOGIN_DISABLED` (setting it to False) undos `TESTING`'s deactivation of the login/authentication processes.